tests/drivers/sensor/accel/src/main.c

```diff
@@ -45,11 +46,25 @@ void test_sensor_accel_basic(void)
 	}
 }
 
-/* test case main entry */
-void test_main(void)
+/* Run all of our tests on an accelerometer device with the given label */
+static void run_tests_on_accel(const char *label)
 {
+	const struct device *accel = device_get_binding(label);
+
+	zassert_not_null(accel, "Unable to get Accelerometer device");
+	k_object_access_grant(accel, k_current_get());
+	accel_label = label;
 	ztest_test_suite(test_sensor_accel,
 			 ztest_user_unit_test(test_sensor_accel_basic));
-
 	ztest_run_test_suite(test_sensor_accel);
 }
+
+/* test case main entry */
+void test_main(void)
+{
+	run_tests_on_accel(DT_LABEL(DT_ALIAS(accel_0)));
+
+#ifdef DT_N_ALIAS_accel_1
```

Comment:
```suggestion
#ifdef DT_ALIAS(accel_1)
```
Use the devicetree.h macros to generate this name for you, right?

Couldn't you even use `IS_ENABLED(DT_ALIAS(accel1))`?

Reply:
I can't use #ifdef on an expression. This is not a CONFIG so IS_ENABLED() doesn't work. I am basically checking for the presence of an alias in the devicetree_unfixed.h file:

/* Existence and alternate IDs: */
#define DT_N_S_i2c_100_S_bmi_68_EXISTS 1
#define DT_N_ALIAS_accel_1       DT_N_S_i2c_100_S_bmi_68
#define DT_N_INST_0_bosch_bmi160 DT_N_S_i2c_100_S_bmi_68
#define DT_N_NODELABEL_bmi_i2c   DT_N_S_i2c_100_S_bmi_68